docs/source/tutorials/own-data.rst

```diff
@@ -309,8 +230,8 @@ Because the above command specifies the ``--num-parts`` to be ``1``, there is on
 
 .. note::
 
-    - Because the parquet format has some limitations, such as only supporting 2 billion elements in a column, etc, we suggest users to use HDF5 format for very large dataset.
-    - The two mapping files, ``node_mapping.pt`` and ``edge_mapping.pt``, are used to record the mapping between the ogriginal node and edge ids in the raw data files and the ids of nodes and edges in the constructed graph. They are important for mapping the training and inference outputs. Therefore, DO NOT move or delete them.
+    - Because the parquet format has some limitations, such as only supporting 2 billion elements in a column, etc, we suggest users to use HDF5 format for very large datasets.
+    - The two mapping files, ``node_mapping.pt`` and ``edge_mapping.pt``, are used to record the mapping between the ogriginal node and edge ids in the raw data files and the ids of nodes and edges in the GraphStorm node ID space. They are important for mapping the training and inference outputs back to the original input data. Therefore, **DO NOT** move or delete them.
```

Comment:
Graph node ID space. Let align the naming with https://github.com/awslabs/graphstorm/pull/621

Reply:
Sure, changed.